Dockerfiles/syslog-ng/syslog-ng.conf

```diff
@@ -0,0 +1,18 @@
+@version: 3.6
+
+source s_net {
+  tcp(
+     ip("127.0.0.1")
+     port(1111)
```

Comment:
Why the port 1111? I'd use a syslog source or a tcp without ports (default ports should do).